### PR TITLE
bunch of new features

### PR DESCRIPTION
**Pull request by domjngo**
Check stuff

### CODE REVIEW

File: page-black-history-landing.php [hunk 1]
```diff
@@ -4,55 +4,201 @@
...
-	<div class="container">
-		<div class="row">
-			<section>
-				<div class="col-md-12">
-					<article>
-						<div class="entry-header">
-							<h2>Black History</h2>
+	<main id="main" role="main" class="content-area">
+		<div class="container">
+			<?php if ( get_post()->post_content ) { ?>
```
Comment (punalsc) [inline page-black-history-landing.php:54]: The code here is fine, but i would tend to use the alternative syntax, as you have alot of HTML embedded in between your conditional statement. Which would be somehthing like this

``` php
<?php if ( get_post()->post_content ) : ?>
      <html>
<?php endif; ?>
```

http://php.net/manual/en/control-structures.alternative-syntax.php

But i would only use this syntax if there is a lot of HTML. @gtvj please share your thoughts.

File: page-black-history-landing.php [hunk 2]
```diff
@@ -4,55 +4,201 @@
...
+			<?php if ( $first_section_heading ) { ?>
+			<section id="first-content-section">
+				<div class="row">
+					<div class="col-md-12">
+						<article>
+							<div class="entry-header">
+								<h2><?php echo $first_section_heading ?></h2>
+							</div>
+							<div class="entry-content clearfix">
+								<?php
```
Comment (punalsc) [inline page-black-history-landing.php:104]: Again, same thing as above, i would use the alternate syntax.

``` php
<?php for ($i = 1; $i <= 3; $i++) : ?> //Start the loop
    <?php //All variables declared
        $title = get_post_meta( $post->ID, 'sec_1_title_'.$i, true );
        $link = get_post_meta( $post->ID, 'sec_1_link_'.$i, true );
        $image = get_post_meta( $post->ID, 'sec_1_image_'.$i, true );
        $description = get_post_meta( $post->ID, 'sec_1_description_'.$i, true );
    ?>
    <?php if ( $title ) : ?>
        <html>
    <?php endif; ?>
<?php endfor; ?>// End the loop
```

**Comment by punalsc** [comment]
Everything checkouts well, only just a few recommendations.
@domingobishop i'm happy to merge the branch into develop if you are happy with it.

Let me know.

File: page-black-history-landing.php (hunk 1 above)
Comment (domjngo) [reply page-black-history-landing.php:54]: I actually prefer curly braces as they are standard block delimiters across many languages and therefore are in general easier to read.

Comment (domjngo) [reply page-black-history-landing.php:54]: And braces are a better syntax for highlighting ( i.e. click opening bracket and it highlights closing bracket ). It's just a preference.

Comment (gtvj) [reply page-black-history-landing.php:54]: Of the two control structure syntax my understanding is that the alternative exists for circumstances like this (with the curly braces being better suited to PHP only code). Opinion is divided about it though - see http://stackoverflow.com/questions/564130/difference-between-if-and-if-endif (My initial thought is that `endif;` helps with readability but I accept Chris' point about highlighting the matching brace. 

That said, the WordPress code style doesn't dictate either https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style 

We should decide on one and use it though - for consistency across our templates. Let's decide on it as a team (I'll put something on Slack).

Comment (gtvj) [reply page-black-history-landing.php:54]: Let's go with the braces syntax`{}`

**Pull request merged**